pages/docs/installation/installation-special-systems.md

```diff
@@ -792,3 +792,26 @@ export PRECICE_BOOST_ROOT=$HOME/software/boost_1_53_0
 * configure ALYA with `-L[PathToPreCICE]/build/last -lprecice -lstdc++ -lrt`
 * for running: also put `module load intel` in your jobscript
 * use `network="ib0"` for sockets communication beyond one node
+
+### Max Planck Computing and Data Facility (MPCDF) Cobra cluster
+
+#### Installing dependencies
+
+##### Eigen3
+
+See the [Eigen dependency section](https://precice.org/installation-source-dependencies.html#eigen) as Eigen is not available as a module on this cluster.
+
+#### Installing preCICE
+
+On the Cobra cluster preCICE is installed from source. Please set the installation prefix paths as shown [in this section](https://precice.org/installation-source-preparation.html#installation-prefix) before installing preCICE. The preCICE repository is either cloned or copied to the cluster, and then the following commands are run:
```

Comment:
Very nice overall, sorry that I comment again on this:
- who is installing preCICE?
- who is either cloning or copying the repository to the cluster?
- who is running the following commands?

I really want to convince you that active voice works better here, but I am no language expert. :sweat_smile: 
Note also that you are mixing passive + active + again passive right now.

```suggestion
On the Cobra cluster, you can easily install preCICE from source. Clone the repository or copy the code to the cluster, set the installation prefix paths as shown [in this section](https://precice.org/installation-source-preparation.html#installation-prefix), and then run the following commands:
```

It would be even better to also comment on what these commands do / why these commands / why these versions. But we don't need too much here.

Reply:
Regarding the mix-up of active and passive voice I concede that its just how I think and hence how I write it, but a consistent voice would be better. As for who the installation guide addresses, I personally find it more elegant to write statement which explain things as they are, rather than addressing the user at all times. In my opinion this is again a matter of personal choice, I go with your choice here.

> It would be even better to also comment on what these commands do / why these commands / why these versions. But we don't need too much here.

The versions of the modules are used just because they work and because only a few versions of all software packages are available. I think it would be difficult to *justify* versions.

Reply:
Just to clarify: this is not just my choice, but a decision we took a few months ago, documented here: https://precice.org/docs-meta-overview.html#language--style

There are several good reasons we decided for the active voice and I will be more than happy to discuss this in person, in the first opportunity. :-)

Reply:
Oh I did not know that we have decided on a style! Thanks for pointing me to it, then I stick to it :smile: